WireDomain/Sources/WireDomain/Repositories/User/UserRepository.swift

```diff
@@ -89,8 +132,75 @@ public final class UserRepository: UserRepositoryProtocol {
         }
     }
 
+    public func updateUser(
+        from event: UserUpdateEvent
+    ) async throws {
+        try await context.perform { [self] in
+
+            let user = fetchOrCreateUser(
+                with: event.id
+            )
+
+            if isFederationEnabled {
+                user.remoteIdentifier = event.userID.uuid
+                user.domain = event.userID.domain
+            }
```

Comment:
question: isn't it only the domain that we should set ?
```suggestion
            user.remoteIdentifier = event.userID.uuid
            if isFederationEnabled {
               user.domain = event.userID.domain
            }
```

Reply:
yes, fixed it